libexec/alluxio-config.sh

```diff
@@ -65,9 +65,17 @@ if [[ -n "${ALLUXIO_LOGS_DIR}" ]]; then
   ALLUXIO_JAVA_OPTS+=" -Dalluxio.logs.dir=${ALLUXIO_LOGS_DIR}"
 fi
 
-if [[ -n "${ALLUXIO_RAM_FOLDER}" ]]; then
-  ALLUXIO_JAVA_OPTS+=" -Dalluxio.worker.tieredstore.level0.dirs.path=${ALLUXIO_RAM_FOLDER}"
+if [[ -z "${ALLUXIO_RAM_FOLDER}" ]]; then
+    if [[ $(uname -s) == Darwin ]]; then
+        # Assuming Mac OS X
+        ALLUXIO_RAM_FOLDER="/Volumes/ramdisk"
+    else
+        # Assuming Linux
+        ALLUXIO_RAM_FOLDER="/mnt/ramdisk"
+    fi
 fi
+ALLUXIO_LEVEL0_DIR_PATH="${ALLUXIO_LEVEL0_DIR_PATH:-${ALLUXIO_RAM_FOLDER}}"
```

Comment:
we need this extra variable so that users are able to configure `alluxio.worker.tieredstore.level0.dirs.path`